docs/navigation-configuration.md

```diff
@@ -66,6 +74,23 @@ You can configure the way Luigi tackles routing in your application in the **Rou
 - **nodeAccessibilityResolver** allows you to define a permission checker function that gets executed on every node. If it returns `false`, Luigi removes the node and its children from the navigation structure.
   **nodeAccessibilityResolver** receives all values defined in the node configuration. See [angular sampleconfig.js](../core/examples/luigi-sample-angular/src/assets/extendedConfiguration.js) for the **constraints** example.
 
+## Context Switcher
+
+Allows to switch between a curated list of navigation elements. Adds a drop down switcher next to the profile in the top navigation bar. Typically used for switching between environments. 
+
+- **emptyLabel** specifies the default label that is shown if no context is selected.
+- **parentNodePath** specifies the base path, that is prepended to **options[].pathValue**. Must be an absolute path.
+- **lazyloadOptions** defines when to fetch **options**. When set to `true`, loads **options** on Context Switcher click without any caching. When set to `false`, loads **options** once on page load . Defaults to `true`. 
+- **options** defines the list of context element. Context element properties are:
+  - **label** defines the context element label. If it is not defined, **pathValue** will be passed to **fallbackLabelResolver** to set its value. Defaults to **pathValue**, if **fallbackLabelResolver** is not defined.
```

Comment:
` If not defined, the  **pathValue** is passed to **fallbackLabelResolver** to set its value. The default value is **pathValue**, if **fallbackLabelResolver** is not defined.`